Bigtable/tests/System/BackupTests.php

```diff
@@ -0,0 +1,203 @@
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable\Tests\System;
+
+use Google\Cloud\Bigtable\Admin\V2\Backup;
+use Google\Protobuf\Timestamp;
+use Google\ApiCore\ApiException;
+
+/**
+ * @group bigtable
+ * @group bigatableBackup
+ */
+class BackupTests extends BigtableTestCase
+{
+    private static $destinationClusterId;
+    private static $destinationProjectId;
+    private static $backupId;
+    private static $backupName;
+    private static $copyBackupId;
+    private static $expireTime;
+
+    public const DESTINATION_LOCATION_ID = 'us-central1-b';
+
+    public static function set_up_before_class()
+    {
+        parent::set_up_before_class();
+        self::$destinationClusterId = uniqid(self::CLUSTER_ID_PREFIX);
+        $keyFilePath = getenv('GOOGLE_CLOUD_PHP_WHITELIST_TESTS_KEY_PATH');
+        $keyFileData = json_decode(file_get_contents($keyFilePath), true);
+        self::$destinationProjectId = $keyFileData['project_id'];
+        self::$backupId = uniqid('backup');
+        self::$copyBackupId = 'copy' . self::$backupId;
+        self::$expireTime = new Timestamp();
+        self::$expireTime->setSeconds(time() + 86400);
+        self::$backupName = self::$tableAdminClient->backupName(
+            self::$projectId,
+            self::$instanceId,
+            self::$clusterId,
+            self::$backupId
+        );
+    }
+
+    public function testCreateBackup()
+    {
+        $backup = new Backup();
+
+        $sourceTableName = self::$tableAdminClient->tableName(
+            self::$projectId,
+            self::$instanceId,
+            self::TABLE_ID
+        );
+        $backup->setSourceTable($sourceTableName);
+        $backup->setExpireTime(self::$expireTime);
+
+        $parentName = self::$instanceAdminClient->clusterName(
+            self::$projectId,
+            self::$instanceId,
+            self::$clusterId
+        );
+        $operationResponse = self::$tableAdminClient->createBackup(
+            $parentName,
+            self::$backupId,
+            $backup
+        );
+        $operationResponse->pollUntilComplete();
+        $result = $operationResponse->getResult();
+        self::assertStringContainsString(self::$backupName, $result->getName());
+        self::assertEquals("", $result->getSourceBackup());
+    }
+
+    /**
+     * @depends testCreateBackup
+     */
+    public function testCopyBackup()
+    {
+        // Test for copying to the same project but different cluster region
+        self::createCluster(
+            self::$projectId,
+            self::$instanceId,
+            self::DESTINATION_LOCATION_ID,
+            self::$destinationClusterId
+        );
+
+        $parent = self::$tableAdminClient->clusterName(
+            self::$projectId,
+            self::$instanceId,
+            self::$destinationClusterId
+        );
+        $operationResponse = self::$tableAdminClient->copyBackup(
```

Comment:
This method does not exist, needs further code changes?

Reply:
This will be auto generated from the [proto changes](https://github.com/googleapis/google-cloud-php/pull/5651#:~:text=in%20preview%20branch.-,Proto,-changes%20/%20additions)